code/modules/research/designs/design_weapons.dm

```diff
@@ -223,12 +233,18 @@ S	fuel tank (hydrazine)
 	build_path = /obj/item/ammo_magazine/pulse
 	price = 1250
 
+/datum/design/item/ammo/pulsedeflect
+	name = "deflection pulse rounds"
+	id = "pulsedeflect"
+	materials = list(MATERIAL_PLASTEEL = 5000, MATERIAL_DIAMOND = 2000, MATERIAL_PHORON = 500)
+	build_path = /obj/item/ammo_magazine/pulse/df
+	price = 4000
+
 /datum/design/item/ammo/pulsehighvel
 	name = "high velocity pulse rounds"
 	id = "pulsehighvel"
-	materials = list(MATERIAL_PLASTEEL = 7500, MATERIAL_DIAMOND = 2500, MATERIAL_PHORON = 500)
+	materials = list(MATERIAL_PLASTEEL = 7500, MATERIAL_DIAMOND = 3000, MATERIAL_PHORON = 500)
 	build_path = /obj/item/ammo_magazine/pulse/hv
-	price = 5750
```

Comment:
In order to remove something from the store edit `build_type` var

Reply:
![image](https://user-images.githubusercontent.com/28861083/167679238-fbc84eb1-40a6-42bb-9699-74ad0b60f838.png)
This work?